### PR TITLE
Add Ever Oasis

### PR DESCRIPTION
**Pull request by bkoss219**
Adds Ever Oasis.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by TheKoopaKingdom** [comment]
Your metadata has no [testcases](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md#metadata), please add one.

**Comment by chris062689** [comment]
You need to convert the files to real PNGs, not just rename JPEG files to PNG.

File: ever-oasis/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Ever Oasis"
+description = "The desert's last safe haven is a lone oasis you call home. So draw a line in the sand and fight to protect it! As you explore the desert and solve puzzling dungeons, you'll gather resources to build up your oasis with the water spirit Esna. Allies will join your cause, so equip them with weapons, and use their skills to slay towering bosses."
+needs_system_files = false
```
Comment (TheKoopaKingdom) [inline ever-oasis/game.dat]: According to [this](https://github.com/citra-emu/citra-games-wiki/pull/44/files#diff-2c3f01ec17ba0112bbb7af6961bbc379R13) line, the game doesn't progress past the Start Screen, so you should add the GitHub issues for that.

File: ever-oasis/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+needs_shared_font = false
+
```
Comment (TheKoopaKingdom) [inline ever-oasis/game.dat]: There's no game type here.

**Pull request closed without merging**